Script/AtomicWebViewEditor/clientExtensions/ClientExtensionServices.ts

```diff
@@ -35,10 +35,23 @@ interface EventSubscription {
 export class EventDispatcher implements Editor.Extensions.EventDispatcher {
     private subscriptions: EventSubscription[] = [];
 
-    sendEvent(eventType: string, data: any) {
+    sendEvent<T extends Atomic.EventCallbackMetaData>(eventCallbackMetaData:T)
+    sendEvent(eventType: string, data: any)
+    sendEvent(eventTypeOrWrapped: any, data?: any) {
```

Comment:
This is just a declaration so that the browser compiles since it's sharing some objects with the editor.  No real magic here. :)